janitor/io.py

```diff
@@ -132,91 +134,412 @@ def xlsx_table(
 
     Example:
 
-    ```python
-
-    filename = "excel_table.xlsx"
-
-    # single table
-    jn.xlsx_table(filename, sheetname='Tables', table = 'dCategory')
-
-        CategoryID      Category
-    0           1       Beginner
-    1           2       Advanced
-    2           3      Freestyle
-    3           4    Competition
-    4           5  Long Distance
-
-    # multiple tables:
-    jn.xlsx_table(filename, sheetname = 'Tables', table = ['dCategory', 'dSupplier'])
-
-    {'dCategory':    CategoryID       Category
-                    0           1       Beginner
-                    1           2       Advanced
-                    2           3      Freestyle
-                    3           4    Competition
-                    4           5  Long Distance,
-    'dSupplier':   SupplierID             Supplier        City State                         E-mail
-                    0         GB       Gel Boomerangs     Oakland    CA          gel@gel-boomerang.com
-                    1         CO  Colorado Boomerangs    Gunnison    CO  Pollock@coloradoboomerang.com
-                    2         CC        Channel Craft    Richland    WA                    Dino@CC.com
-                    3         DB        Darnell Booms  Burlington    VT            Darnell@Darnell.com}
-    ```
-
-    :param path: Path to the Excel File.
+        >>> import pandas as pd
+        >>> from janitor import xlsx_table
+        >>> filename = "../pyjanitor/tests/test_data/016-MSPTDA-Excel.xlsx"
+
+        # single table
+        >>> xlsx_table(filename, sheetname='Tables', table = 'dCategory')
+           CategoryID       Category
+        0           1       Beginner
+        1           2       Advanced
+        2           3      Freestyle
+        3           4    Competition
+        4           5  Long Distance
+
+        # multiple tables:
+        >>> out = xlsx_table(filename, sheetname = 'Tables', table = ['dCategory', 'dSalesReps'])
+        >>> out['dCategory']
+           CategoryID       Category
+        0           1       Beginner
+        1           2       Advanced
+        2           3      Freestyle
+        3           4    Competition
+        4           5  Long Distance
+        >>> out['dSalesReps'].head(3)
+           SalesRepID             SalesRep Region
+        0           1  Sioux Radcoolinator     NW
+        1           2        Tyrone Smithe     NE
+        2           3         Chantel Zoya     SW
+
+
+    :param path: Path to the Excel File. It can also be an openpyxl Workbook.
```

Comment:
would appreciate your advice on how to go about that, since at the moment we are importing the workbook in a try clause, which triggers an error message if openpyxl is not installed/not found